core/da/client.go

```diff
@@ -14,6 +14,15 @@ type Client interface {
 
 	// MaxBlobSize returns the maximum blob size for the DA layer.
 	MaxBlobSize(ctx context.Context) (uint64, error)
+
+	// SubmitBatch submits a batch of blobs to the DA layer.
+	SubmitBatch(ctx context.Context, data [][]byte, maxBlobSize uint64, gasPrice float64) ResultSubmitBatch
```

Comment:
can we just use `Submit` instead of `SubmitHeaders` and `SubmitBatch`? Same for retrieve. There is no difference in the code right?

Reply:
yes, was thinking the same, will adjust